lib/vagrant-service-manager/command.rb

```diff
@@ -183,7 +183,7 @@ def restart_service(service)
                     # to OpenShift service
                     "sccli #{service}"
```

Comment:
Agree on it. 
@praveenkumar : Any possibility to have such feature in `sccli` in future?